package/MDAnalysis/analysis/pca.py

```diff
@@ -415,7 +417,15 @@ def transform(self, atomgroup, n_components=None, start=None, stop=None,
 
         dot = np.zeros((n_frames, dim))
 
-        for i, ts in enumerate(traj[start:stop:step]):
+        if verbose == True:
+            self.disabled = False
+        else:
+            self.disabled = True
```

Comment:
Why do you need to store the value in the new attribute `self.disabled`? 

I think you'll only ever use it in this function. 

So unless there are really important reasons (please explain!), remove `self.disabled`.

Reply:
I was initially going to check if there were any other open issues for adding ProgressBar/verbose option in PCA, so I saved it so we can reuse it if needed, but you are right, it doesn't seem so. Removed now.